### PR TITLE
Fix README.md

### LINKED ISSUE #6: [Bug] Cannot pull the image "docker.pkg.github.com/YOURNAME/js-build/tic-tac-toe:f29"
**Describe the bug**
Cannot pull the docker image when following instructions.

**To Reproduce**
1. Follow instructions and perform activities (Steps 1 to 3), so that you merge the pull request.
2. In step 4, log in to Docker using a personal access token.
3. Type the following command: `docker pull docker.pkg.github.com/YOURNAME/js-build/tic-tac-toe:f29`.
4. Press **Enter** and you will see an error in the output.

**Expected behavior**
According to README.md, you should see output indicating that the pull was successful, like Status: Downloaded newer image for docker.
However the output says, "Error response from daemon: manifest unknown".

**Screenshots**

**Device information**
 - Type: Desktop PC
 - OS: Windows 11
 - Browser: Chrome
 - Version: 103.0.5060.114

**Additional context**

### LINKED ISSUE #8: [Feature]Change registry to GHCR
**Is your feature request related to a problem? Please describe.**
[docker.pkg.github.com](https://docker.pkg.github.com) is deprecated. Use [GHCR](https://ghcr.io) instead.

**Describe the solution you'd like**
Use GHCR in this course.

**Describe alternatives you've considered**

**Additional context**
Related to #6

### PR DESCRIPTION
**Pull request by sinsukehlab**
Change the docker package to pull in Step 4

### Why:

Closes #6, Closes #8

### What's being changed:

I have changed some instructions mainly in Step 4 so that learners can pull the correct Docker image.
~~:warning: **Some screenshots need changing.**~~

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ First, take a moment to examine the image below. It shows the relationship betwe
...
 
 [**Docker**](https://www.docker.com/why-docker) is an engine that allows you to run containers.
-Containers are packages of software that can run reliably in different environments. Containers include everything needed to run the application. Containers are lightweight in comparison to virtual machines. A **dockerfile** is a text document that contains all the commands and instructions necessary to build a Docker Image. A **Docker image** is an executable package comprised of code, dependancies, libraries, a runtime, environment variables, and configuration files. A **Docker container** is a runtime instance of a Docker Image.
+Containers are packages of software that can run reliably in different environments. Containers include everything needed to run the application. Containers are lightweight in comparison to virtual machines. A **Dockerfile** is a text document that contains all the commands and instructions necessary to build a Docker Image. A **Docker image** is an executable package comprised of code, dependancies, libraries, a runtime, environment variables, and configuration files. A **Docker container** is a runtime instance of a Docker Image.
```
Comment (sinsukehlab) [inline README.md:63]: Fix a typo
Cf. #11
```suggestion
Containers are packages of software that can run reliably in different environments. Containers include everything needed to run the application. Containers are lightweight in comparison to virtual machines. A **Dockerfile** is a text document that contains all the commands and instructions necessary to build a Docker Image. A **Docker image** is an executable package comprised of code, dependencies, libraries, a runtime, environment variables, and configuration files. A **Docker container** is a runtime instance of a Docker Image.
```

**Comment by sinsukehlab** [comment]
@heiskr @dihydroJenoxide I would appreciate it if you could review this PR at your earliest convenience.
Replacing those screenshots that I indicated would be an arduous task, but I would be very grateful if you could improve this PR.

**Comment by sinsukehlab** [comment]
I would like to open another PR to close #8 after this PR is merged.

File: README.md (hunk 1 above)
Comment (Sebastianoskiojoj) [reply README.md]: > Fix a typo Cf. #11

**Review by Sebastianoskiojoj — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Sebastianoskiojoj) [reply README.md]: GET {{ GITHUB_API_BASE_URL }}/repos/{{ org }}/{{ repo }}
Accept: application/vnd.github.v3+json
Authorization: token {{ GITHUB_TOKEN }}

HTTP/2 200

[Asserts]
status >= 200
status < 300
header "Content-Type" == "application/json; charset=utf-8"
header "x-github-request-id" isString
jsonpath "$.name" == "{{ repo }}"
jsonpath "$.full_name" == "{{ org}}/{{ repo }}"

**Comment by wongcyrus** [comment]
The instruction should ask user to get the SHA tag from GitHub Action for Activity: Pull your image 

Here is video demo for the step.
[Demo to get the tag from GitHub Action](https://youtu.be/vfxhVIBjMb0?t=925)

**Comment by sinsukehlab** [comment]
> The instruction should ask user to get the SHA tag from GitHub Action for Activity: Pull your image
> 
> Here is video demo for the step. [Demo to get the tag from GitHub Action](https://youtu.be/vfxhVIBjMb0?t=925)

You can get the tag from **Packages** page.

**Comment by sinsukehlab** [comment]
> The instruction should ask user to get the SHA tag from GitHub Action for Activity: Pull your image
> 
> Here is video demo for the step. [Demo to get the tag from GitHub Action](https://youtu.be/vfxhVIBjMb0?t=925)

You can get the tag from the **Packages** heading link.
It redirects to the list of packages you have published.

File: README.md [hunk 2]
```diff
@@ -164,7 +164,7 @@ Before we can use this Docker image, you will need to generate a [personal acces
 - write:packages
 - read:packages
 
-![screenshot personal access token creation page with boxes for repo (all), write:packages, and read:packages checked](https://i.imgur.com/Ue9BJoV.png)
+![screenshot personal access token creation page with boxes for repo (all), write:packages, and read:packages checked](https://i.imgur.com/Ue9BJoV.png)<!-- Now that the box for read:packages is the child of the box for write:packages, this screenshot should be changed. -->
```
Comment (chadfawcett) [inline README.md:167]:
```suggestion
![screenshot personal access token creation page with boxes for repo (all), write:packages, and read:packages checked](https://user-images.githubusercontent.com/3250463/219254714-82bb1da5-33b1-491b-97c0-b25f51494f6a.png)
```
---
## Image:
Screen Shot 2023-02-15 at 6 33 36 PM

File: README.md [hunk 3]
```diff
@@ -182,11 +182,14 @@ If everything went well, 🤞 you should see `Login Succeeded` in your terminal.
 ### :keyboard: Activity: Pull your image
 
 1. Copy and paste the `pull` command from the package instructions into your terminal. It should look something like this:
-   - `docker pull docker.pkg.github.com/YOURNAME/js-build/tic-tac-toe:f29`
-   ![screenshot of the pull command on the GitHub package page](https://i.imgur.com/pFQgfSZ.png)
+   - `docker pull docker.pkg.github.com/YOURNAME/publish-packages/game:TAG`
+   ![screenshot of the pull command on the GitHub package page](https://i.imgur.com/pFQgfSZ.png)<!-- This screenshot should be changed. -->
```
Comment (chadfawcett) [inline README.md:186]:
```suggestion
   ![screenshot of the pull command on the GitHub package page](https://user-images.githubusercontent.com/3250463/219254981-9ff949fa-4d01-46e3-9e3d-b8ce3710c2a9.png)
```

---

## Image:
Screen Shot 2023-02-15 at 6 34 06 PM

File: README.md [hunk 4]
```diff
@@ -182,11 +182,14 @@ If everything went well, 🤞 you should see `Login Succeeded` in your terminal.
...
+   - _Tip: To reach this page, click the **Code** tab at the top of your repository. Then, find the navigation bar below the repository description, and click the **Packages** heading link_
+1. Replace `YOURNAME` with your GitHub username.
+1. Replace `TAG` with the image tag.
 1. Press **Enter**.
-1. You should see output indicating that the pull was successful, like `Status: Downloaded newer image for docker`.
-   ![screenshot of successful Docker image output](https://i.imgur.com/i07kF2J.png)
+1. You should see output indicating that the pull was successful, like `Status: Downloaded newer image for docker.pkg.github.com/YOURNAME/publish-packages/game:TAG`.
+   ![screenshot of successful Docker image output](https://i.imgur.com/i07kF2J.png)<!-- This screenshot should be changed. -->
```
Comment (chadfawcett) [inline README.md:192]:
```suggestion
   ![screenshot of successful Docker image output](https://user-images.githubusercontent.com/3250463/219255178-3c943a6f-6c15-4f59-9002-228249b1c469.png)
```

---

## Image:
Screen Shot 2023-02-15 at 6 24 22 PM

File: README.md [hunk 5]
```diff
@@ -206,15 +209,15 @@ Let's trying running it.
 
 ### :keyboard: Activity: Run your image
 
-1. Find your image information by typing `Docker image ls`.
-   ![screenshot of output from Docker image ls command: lists docker images, REPOSITORY TAG and docker URL](https://i.imgur.com/UAwRXiq.png)
+1. Find your image information by typing `docker image ls`.
+   ![screenshot of output from Docker image ls command: lists docker images, REPOSITORY TAG and docker URL](https://i.imgur.com/UAwRXiq.png)<!-- This screenshot should be changed. -->
```
Comment (chadfawcett) [inline README.md:213]: I have way too many images locally so didn't take a screenshot of this ☹️

File: README.md [hunk 6]
```diff
@@ -206,15 +209,15 @@ Let's trying running it.
...
 1. Use the following command to run a container from your image:
    ```
    docker run -d -it --rm -p 8080:80 --name ttt <YOUR_IMAGE_NAME:TAG>
    ```
 1. Replace `YOUR_IMAGE_NAME` with your image name under the `REPOSITORY` column.
 1. Replace `TAG` with the image tag under the `TAG` column
-   ![example of running the docker command listed above](https://i.imgur.com/hr6N9nk.png)
+   ![example of running the docker command listed above](https://i.imgur.com/hr6N9nk.png)<!-- This screenshot should be changed. -->
```
Comment (chadfawcett) [inline README.md:220]:
```suggestion
   ![example of running the docker command listed above](https://user-images.githubusercontent.com/3250463/219255534-f11b20e8-65de-4f4a-a033-f312ddf507fb.png)
```

---

## Image:
Screen Shot 2023-02-15 at 6 27 01 PM

**Review by chadfawcett — COMMENTED** [review]
@skills/content-reviewers I just went through the course while working on #18. I can confirm these changes are necessary/beneficial. I took some new screenshots and added them as commit suggestions below. Unfortunately I named my repo `tmp-publish-packages`, so it's not exactly what the learner would see, but its at least closer.

File: README.md (hunk 6 above)
Comment (sinsukehlab) [reply README.md:230]: This screenshot does not agree with the instruction.
How would you like the interactive and tty options `-it` and the port option `-p 8080:80`?

**Comment by sinsukehlab** [comment]
@chadfawcett I highly appreciate your suggestions.
I have committed some suggestions. I have also cherry-picked a commit from another branch.

File: README.md (hunk 6 above)
Comment (chadfawcett) [reply README.md:230]: Oh ya, I forgot I tested removing those arguments. My assumption was that those arguments were for the old docker image. They're not needed for this new one. So the example on line 213/216 would be changed to match this screenshot. **IF** my assumption is right.

**Comment by sinsukehlab** [comment]
I have split this PR.
PR #19 changes the YAML code block.

**Comment by heiskr** [comment]
This looks like a useful set of changes, I'll go ahead and merge but feel free to open additional pull requests.

File: README.md (hunk 6 above)
Comment (sinsukehlab) [reply README.md]: Well, the option `-p 8080:80` is actually needed if learners would like to open the page (index.html).
I have opended another PR #22.

**Pull request merged**